### PR TITLE
Update example project .gitignores to latest template

### PR DESCRIPTION
**Pull request by cbracken**
Under android/ directory, ignore:
GeneratedPluginRegistrant.java

Under ios/ directory, ignore:
GeneratedPluginRegistrant.h
GeneratedPluginRegistrant.m
Pods/

### CODE REVIEW

File: examples/flutter_gallery/ios/.gitignore [hunk 1]
```diff
@@ -10,8 +10,8 @@ profile
 
 DerivedData/
 build/
-PluginRegistry.h
-PluginRegistry.m
+GeneratedPluginRegistrant.h
+GeneratedPluginRegistrant.m
 
 *.pbxuser
 *.mode1v3
```
Comment (goderbauer) [inline examples/flutter_gallery/ios/.gitignore]: Ignore Pods?

Comment (goderbauer) [inline examples/flutter_gallery/ios/.gitignore]: Never mind, found it.

Comment (cbracken) [inline examples/flutter_gallery/ios/.gitignore]: Already ignored at the bottom.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**